libtiledbsoma/src/reindexer/test_indexer_dtatye_perf.py

```diff
@@ -0,0 +1,99 @@
+from time import perf_counter
```

Comment:
@beroy is `test_indexed_dtatye_perf.py` a typo? What does `dtatye` mean?

Reply:
It's a typo. It should say `data_types`